app/services/guards/create_bill_run_v2_guard.service.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+/**
+ * @module CreateBillRunV2GuardService
+ */
+
+const Boom = require('@hapi/boom')
+
+class CreateBillRunV2GuardService {
```

Comment:
I freakin ❤️  you hit on the name `Guard`. I think we have a few others fulfilling this role.

I'm just checking in on placement. Previously, where a service has been connected to a thing, for example, `BillRun` it's been placed in the `services/bill_runs` folder.  Have you got a reason for wanting to start a `services/guards` folder as a distinct thing?